### ISSUE TITLE
07-12-23 she's broken again!

### ISSUE THREAD

**Issue by alpha520098** [sam43434/remootio#25]
I see that the aiohttp has gone to 3.9.1 could that be the issue?

@yarafie

**Comment by alpha520098** [comment]
Unable to install package aioremootio @ git+https://github.com/alpha520098/aioremootio.git: ERROR: Cannot install aioremootio==1.0.0.dev0 because these package versions have conflicting dependencies. ERROR: ResolutionImpossible: for help visit https://pip.pypa.io/en/latest/topics/dependency-resolution/#dealing-with-dependency-conflicts

@yarafie

**Comment by alpha520098** [comment]
https://github.com/home-assistant/core/pull/104152

found this i dont know if it will help @yarafie

**Comment by lbreggi** [comment]
same issue for me

**Comment by jwinston100** [comment]
My Remootio integration broke as well with the new HA version, core 2023.12.0 and 2023.12.1. It does seem this is due to the upgraded version of aiohttp, so I updated my requirements.txt in my Github fork as previously described by @yarafie to this:

```
aiohttp>=3.8.5,<4.0
pycryptodome>=3.17
async-class>=0.5.0,==0.5.*
voluptuous>=0.13.1,==0.13.*
```

The integration now works.

**Comment by drsmarsden** [comment]
I cloned the repository to fix the versioning 

https://github.com/drsmarsden/aioremootio and changed  the Requirements string in manifest.json in the Remootio custom component 

I intend to keep this repo working

**Comment by alpha520098** [comment]
> I cloned the repository to fix the versioning
> 
> https://github.com/drsmarsden/aioremootio and changed the Requirements string in manifest.json in the Remootio custom component
> 
> I intend to keep this repo working

Is there a way that this could be a intergation to home assistant? or is there a away that this could auto update?

not that im complaining because i've had @jwinston100 and @peaceduck @yarafie  fix this issue many times but it would be nice to have this auto update?

**Comment by drsmarsden** [comment]
Above my paid grade sorry, you could email the core dev team and ask if it could be included in the core. I think that is based on number of users and adherence to coding standards. Although I have a large HA setup, I am in the “retired tinkerer “ category SorrySent from my iPadOn 14 Dec 2023, at 11:54, alpha520098 ***@***.***> wrote:﻿

I cloned the repository to fix the versioning
https://github.com/drsmarsden/aioremootio and changed the Requirements string in manifest.json in the Remootio custom component
I intend to keep this repo working

Is there a way that this could be a intergation to home assistant? or is there a away that this could auto update?
not that im complaining because i've had @jwinston100 and @peaceduck @yarafie  fix this issue many times but it would be nice to have this auto update?

**Comment by drsmarsden** [comment]
On thing I might try , if it references a core python library, why does Remootio need to also declare it as a dependency Sent from my iPadOn 14 Dec 2023, at 14:22, Stuart Marsden ***@***.***> wrote:﻿Above my paid grade sorry, you could email the core dev team and ask if it could be included in the core. I think that is based on number of users and adherence to coding standards. Although I have a large HA setup, I am in the “retired tinkerer “ category SorrySent from my iPadOn 14 Dec 2023, at 11:54, alpha520098 ***@***.***> wrote:﻿

I cloned the repository to fix the versioning
https://github.com/drsmarsden/aioremootio and changed the Requirements string in manifest.json in the Remootio custom component
I intend to keep this repo working

Is there a way that this could be a intergation to home assistant? or is there a away that this could auto update?
not that im complaining because i've had @jwinston100 and @peaceduck @yarafie  fix this issue many times but it would be nice to have this auto update?

**Comment by yarafie** [comment]
> > I cloned the repository to fix the versioning
> > https://github.com/drsmarsden/aioremootio and changed the Requirements string in manifest.json in the Remootio custom component
> > I intend to keep this repo working
> 
> Is there a way that this could be a intergation to home assistant? or is there a away that this could auto update?
> 
> not that im complaining because i've had @jwinston100 and @peaceduck @yarafie fix this issue many times but it would be nice to have this auto update?

similar to @drsmarsden reply. It is a major undertaking. that i don't have the time nor the inclination to do to make it an officil integration. I also forked the aioremootio and remootio and update as needed  as a HACS. I think there was an attempt by the aioremootio dev but haven't seen much update.

**Comment by peaceduck** [comment]
I have been really busy with work this autumn. But I have not completely given up on taking a closer look at this in the beginning of next year. I would like to get this to be official, so if I can find time for it I will have a go at that. But no promises. It depends on my work situation...

**Comment by yarafie** [comment]
> I have been really busy with work this autumn. But I have not completely given up on taking a closer look at this in the beginning of next year. I would like to get this to be official, so if I can find time for it I will have a go at that. But no promises. It depends on my work situation...

Thanks. I saw by chance  in the HA dev github some attempts by somone to create a remootio integration with a long list of requirements and discussions. was that by chance you? can't seem to find it again.

**Comment by peaceduck** [comment]
No, that was not me. I will try to see if I can find anything on it. Could be a good read, if I can find it. If you happen to find where you saw it, please let me know!

**Comment by yarafie** [comment]
> No, that was not me. I will try to see if I can find anything on it. Could be a good read, if I can find it. If you happen to find where you saw it, please let me know!

I was actually searching for a way to get the remootio icon to appear  in HA ad it is blank. Few hours of googling went down the proverbial rabbit hole :) and stumblefd upon it. Should have bookmarked it tnought i did but can't find it again. I think it is the actual developer of aioremootio ivgg-me if I recall. If i stumble upon it again will let you know.

**Comment by yarafie** [comment]
> No, that was not me. I will try to see if I can find anything on it. Could be a good read, if I can find it. If you happen to find where you saw it, please let me know!
I feel like dorothy :) @peaceduck 
 check below
https://github.com/home-assistant/core/pull/63155

**Comment by drsmarsden** [comment]
Is there a spec of what needs to be done .? I am having surgery  at the start of January so will be house bound for 6 weeks , so only so much piano practice and Netflix you can doSent from my iPhoneOn 14 Dec 2023, at 18:56, yarafie ***@***.***> wrote:﻿

No, that was not me. I will try to see if I can find anything on it. Could be a good read, if I can find it. If you happen to find where you saw it, please let me know!
I feel like dorothy :) @peaceduck
check below
home-assistant/core#63155

**Comment by yarafie** [comment]
> Is there a spec of what needs to be done .? I am having surgery  at the start of January so will be house bound for 6 weeks , so only so much piano practice and Netflix you can doSent from my iPhoneOn 14 Dec 2023, at 18:56, yarafie ***@***.***> wrote:﻿ No, that was not me. I will try to see if I can find anything on it. Could be a good read, if I can find it. If you happen to find where you saw it, please let me know! I feel like dorothy :) @peaceduck check below [home-assistant/core#63155](https://github.com/home-assistant/core/pull/63155) —Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were mentioned.Message ID: ***@***.***>

@drsmarsden no idea. wish you a succesful  surgery and speedy recovery.

**Comment by alpha520098** [comment]
Like wise, hope you have a speedy recovery, however if you wish to put this as a integration in HA @yarafie will give you back rub

**Comment by markatprecision** [comment]
I'd do anyhting (almost) to have this fixed!!!

**Comment by alpha520098** [comment]
Mate you having issues?

**Comment by markatprecision** [comment]
Remootio integration > even changed the manifest file to 
"requirements": ["aioremootio @ git+https://github.com/sam43434/aioremootio.git"],

no joy. Is it working for anyone?
I get no devices and no entities 
![image](https://github.com/sam43434/remootio/assets/92554341/ead2f1a8-36de-46ee-a5d1-2efdd3e946af)

**Comment by alpha520098** [comment]
@markatprecision 

Mate get to this Facebook page which and I'll help you through that 

https://www.facebook.com/profile.php?id=61553050806894&mibextid=LQQJ4d

**Comment by yarafie** [comment]
> @markatprecision
> 
> Mate get to this Facebook page which and I'll help you through that
> 
> https://www.facebook.com/profile.php?id=61553050806894&mibextid=LQQJ4d

Yes. Also if anyones is interrested i just fixed (in my versions in github) the warning messages in HA regarding depcretated SUPPORT_* messages. I confirm it works.

**Comment by alpha520098** [comment]
If I could ask @yarafie to join that page it would very helpful- he's certainly the man when it comes to these issues

**Comment by yarafie** [comment]
> If I could ask @yarafie to join that page it would very helpful- he's certainly the man when it comes to these issues

Not a big fan of facebook but since you kindly asked i just joined.

**Comment by markatprecision** [comment]
I added that one line and it worked, thanks.



Precision Consulting Corporation
PO Box 376, Castle Hill NSW 1765 Australia         |      E-Mail: ***@***.***
Tel: +61 409 227 400                                       |      Web:  Http://PrecisionConsulting.com.au<http://precisionconsulting.com.au/>

Important: This E-mail and attachments are confidential and may be legally privileged. If you are not the intended recipient, you must immediately delete the message and not disclose, copy, forward or use the information contained in it in any way.
